sync-diff-inspector: add annontation for option configuration items (#6959)
pingcap/docs: sync-diff-inspector/sync-diff-inspector-overview.md

- `Global config`: General configurations, including number of threads to check, whether to export SQL statement to fix inconsistent tables, whether to campare the data and so on.
- `Databases config`: Configures the instances of the upstream and downstream databases.
- `Routes`: Rules for upstream multiple schema names to match downstream single schema names **(optional)**.
- `Task config`: Configures the tables for checking. If some tables have a certain mapping relationship between the upstream and downstream databases or have some special requirements, you must configure these tables.

> - `Table config`: Special configurations for specific tables, including specified ranges, columns to be ignored and so on **(optional)**.

Reviewer: ```suggestion
- `Table config`: Special configurations for specific tables, such as specified ranges and columns to be ignored **(optional)**.
```
including一般用于穷举。这里用such as比较好，而且用such as后面也不能跟 and so on

######################### Datasource config #########################
[data-sources]

> [data-sources.mysql1] # mysql1 is the only custom ID for the database instance. It is used for following `task.source-instances/task.target-instance` configuration.

Reviewer: ```suggestion
[data-sources.mysql1] # mysql1 is the only custom ID for the database instance. It is used for the following `task.source-instances/task.target-instance` configuration.
```
